tests/test_pipeline.py

```
@@ -103,3 +103,61 @@ def test_invalid_step_kwargs(self, sample_csv):
+        assert result.columns == ["full_name"]
+        assert ar.to_pandas(result)["full_name"].iloc[0] == "Ada Lovelace"
+
+    def test_combine_columns_rejects_empty_columns(self, sample_csv):
+        frame = ar.read_csv(sample_csv)
+
+        try:
+            ar.pipeline(
+                frame,
+                [("combine_columns", {"columns": [], "output_column": "combined"})],
+            )
+            assert False, "Should have raised ValueError"
```

Ruff (0.15.12)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)